From 029e18901ccc08f48888d906c2479594e8831a43 Mon Sep 17 00:00:00 2001
From: Brian McKenna <bmckenna@atlassian.com>
Date: Mon, 12 Oct 2015 13:20:43 +1100
Subject: [PATCH] Add PINCTRL back in as a menu option

This is to help with make oldconfig and the NixOS generate-config.pl script.
---
 drivers/pinctrl/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/Kconfig b/drivers/pinctrl/Kconfig
index 84dd2ed..c3e6604 100644
--- a/drivers/pinctrl/Kconfig
+++ b/drivers/pinctrl/Kconfig
@@ -3,7 +3,7 @@
 #
 
 config PINCTRL
-	bool
+	bool "PINCTRL"
 
 if PINCTRL
 
-- 
2.5.2

